README.md

```diff
@@ -1,2 +1,180 @@
-# AutoAI-code-generation
-code pattern to show notebook generation feature of AutoAI
+
+<!-- Add a new Title and fill in the blanks -->
```

Comment:
please delete these  comments when you are done with them.  Yes I sometimes keep them i my drafts, but eventually it becomes annoying clutter.